tools/clang/test/CodeGenHLSL/lib_no_alias.hlsl

```diff
@@ -1,7 +1,7 @@
 // RUN: %dxc -T lib_6_3 -auto-binding-space 11 -default-linkage external %s | FileCheck %s
 
 // Make sure out param has no-alias.
-// CHECK: float @"\01?test@@YAMMUT@@AIAV?$matrix@M$01$01@@M@Z"(float %a, %struct.T* noalias nocapture %t, %class.matrix.float.2.2* noalias nocapture dereferenceable(16) %m, float %b)
+// CHECK: float @"\01?test{{[@$?.A-Za-z0-9_]+}}"(float %a, %struct.T* noalias nocapture %t, %class.matrix.float.2.2* noalias nocapture dereferenceable(16) %m, float %b)
```

Comment:
Will the \01? at begging of function name change in future mangling?

Reply:
No, that's part of the Microsoft ABI and we won't be getting rid of that just extending it.

Reply:
Would it make more sense to use '*' (0 or more) instead of '+'? Or will name mangling always exist in some form or another?